.travis.yml

```diff
@@ -1,14 +1,16 @@
 ---
+dist: xenial
 language: python
-python: "2.7.13"
+python: "3.7"
 sudo: required
 services:
   - docker
 install:
   - pip install pipenv
-  - pipenv install -r test-requirements.txt --python 2.7
+  - pipenv sync
 script:
-  - pipenv run molecule test
+  - pipenv run molecule test --scenario-name=mysql
```

Comment:
why not `pipenv run molecule test --all`? This way, if any new scenario is added, there'll be no need of updating this file :)